README.md

```diff
@@ -1 +1,33 @@
-# solution-template-repository
\ No newline at end of file
+# Release Information
+
+- **Version**:  1.0.0
+- **Certified**: Yes
+- **Publisher**: Fortinet
+- **Compatible Version**: FortiSOAR v7.6.1 and later
+
+# Overview
+
+**Unleash the Power of Threat Intelligence with Our FortiGuard Labs IOC Search Solution Pack**
+
+Empower your SOC analysts with unparalleled visibility into malicious indicators using the FortiGuard Labs IOC Search Solution Pack. This innovative solution seamlessly integrates the powerful capabilities of FortiGuard Labs into FortiSOAR, delivering actionable threat intelligence at your fingertips.
```

Comment:
lets try to avoid using solution packs as its just a packaging mechanism. Instead we can treat this as a feature.
Also, this feature will ne consumed in FSR as well as in FTIP, hence mention of FSR will not be advisable here. Can we keep it generic by any means?

Reply:
You can add some screenshots in this page as well

Reply:
Screenshot already present.
Removed references to "solution pack"